scripts/release.py

```
@@ -173,30 +174,30 @@ def main():
             new_version = increment_version(current_version, "minor")
         elif args.next_major:
             new_version = increment_version(current_version, "major")
-        
+
         print(f"New version: {new_version}")
```

Pylint (3.3.7)

[error] Possibly using variable 'new_version' before assignment

(E0606)